components/compiler-families/arm-compilers-devel/SPECS/arm-compilers-devel.spec

```diff
@@ -0,0 +1,159 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+%{!?PROJ_DELIM: %global PROJ_DELIM -ohpc}
+
+%define pname arm-compilers-devel
```

Comment:
I prefer using %global to avoid scope issues

Reply:
fixed.